integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -284,6 +285,10 @@ def move(self, target, pos=None):
         invalidate_model(self.__class__)
         super().move(target=target, pos=pos)
         invalidate_model(self.__class__)
+        if self.depth == 1:
+            # Moving a complete tree causes a tree_id change in the page model
+            # so the cache of the page translation - page relation has to be cleared
+            invalidate_model(PageTranslation)
```

Comment:
This model is also used as base for the language tree node, so I think we should move this invalidation to the page model.
Also, I think just to be 100% save we should invalidate the translation model in any case - even though the `tree_id` does not change, the `lft` and `rgt` values change which might also cause caching issues in some weird edge cases.

Reply:
Thank you very much for your review, you are right, I moved the invalidation to the page model